entity-service/build.gradle.kts

```diff
@@ -34,6 +36,9 @@ tasks.bootRun {
 
 jib.from.image = project.ext["jibFromImage"].toString()
 jib.to.image = "stellio/stellio-entity-service"
-jib.container.jvmFlags = listOf(project.ext["jibContainerJvmFlag"].toString())
+jib.container.entrypoint = listOf("/bin/sh")
```

Comment:
and last point, why splitting it in entrypoint and args and not putting all in entrypoint ?